pages/500.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Page } from '@/components/Page/Page'
+
+export default function error500()
+{
+  return (
+    <Page>
+      <section className="h-screen p-16 flex flex-col px-5 mx-auto justify-center items-center">
```

Comment:
if i make the screen size smaller, the texts aren't centered and the x axis margin is a bit off in some screen sizes

Reply:
i still see this
<img width="350" alt="Screen Shot 2022-07-24 at 2 01 14 PM" src="https://user-images.githubusercontent.com/62922309/180665712-a13fc222-48d9-4928-852b-05396bb020c0.png">

Reply:
I think you need `text-center`